agent.go

```diff
@@ -261,7 +276,14 @@ func (agent *agentS) fullRequestResponse(ctx context.Context, url string, method
 
 	if err == nil {
 		if j != nil {
-			req, err = http.NewRequest(method, url, bytes.NewBuffer(j))
+			b := bytes.NewBuffer(j)
+			if b.Len() > maxContentLength {
+				sensor.logger.Warn(`A batch of spans have been rejected because they are too large to be sent to the agent.`)
```

Comment:
*nitpick:* The verb refers to _the batch_, which is singular. 🤓  

```suggestion
				sensor.logger.Warn(`A batch of spans has been rejected because it is too large to be sent to the agent.`)
```

Reply:
Nice catch 👍 Thank you!